website/docs/indexing.md

```diff
@@ -155,4 +155,11 @@ to finally check the incoming updates against all files. The `SIMPLE` Index will
 `HBASE` index can be employed, if the operational overhead is acceptable and would provide much better lookup times for these tables.
 
 When using a global index, users should also consider setting `hoodie.bloom.index.update.partition.path=true` or `hoodie.simple.index.update.partition.path=true` to deal with cases where the
-partition path value could change due to an update e.g users table partitioned by home city; user relocates to a different city. These tables are also excellent candidates for the Merge-On-Read table type.
\ No newline at end of file
+partition path value could change due to an update e.g users table partitioned by home city; user relocates to a different city. These tables are also excellent candidates for the Merge-On-Read table type.
+
+
+## Related Resources
+<h3>Videos</h3>
+
+* [Global Bloom Index: Remove duplicates & guarantee uniquness - Hudi Labs](https://youtu.be/XlRvMFJ7g9c)
+* [Advantages of Metadata Indexing and Asynchronous Indexing in Hudi - Hands on Lab](https://www.youtube.com/watch?v=TSphQCsY4pY)
```

Comment:
This one is a good fit for the page - https://hudi.apache.org/docs/next/metadata_indexing

Reply:
Lets move it over to there.